src/AppInstallerRepositoryCore/Microsoft/Schema/1_0/Interface_1_0.cpp

```diff
@@ -58,7 +58,7 @@ namespace AppInstaller::Repository::Microsoft::Schema::V1_0
         }
 
         // Gets a manifest id by the given key values.
-        std::optional<SQLite::rowid_t> GetManifestIdByKey(SQLite::Connection& connection, SQLite::rowid_t id, std::string_view version = "", std::string_view channel = "")
+        std::optional<SQLite::rowid_t> StaticGetManifestIdByKey(const SQLite::Connection& connection, SQLite::rowid_t id, std::string_view version = "", std::string_view channel = "")

```

Comment:
>StaticGetManifestIdByKey [](start = 39, length = 24)

Curious why this refactoring? I didn't see it's used in other places other than the original GetManifestIdByKey

Reply:
The name change? Or why is it in it's own function? I changed the name rather than figure out how to reference it from the member function that now has the same name.  As to why it wasn't just in the other function, I don't know.  But I see no reason to move it at this time.

---
In reply to: [492439957](https://github.com/microsoft/winget-cli/pull/575#discussion_r492439957) [](ancestors = 492439957)